reviews/methods1.Rmd

```diff
@@ -0,0 +1,450 @@
+The authors say they provide a 'Bayesian rationalization'.
+This is not correct.
+
+> We respectfully disagree.
+> What makes the arguments "Bayesian" is what we take probabilities to mean.
+> For a Bayesian, they represent degrees of belief.
+> For a frequentist, they represent (relative) frequencies of events that result from a random process.
+> This is a subtle but most important distinction.
+
+Their rationalization is in essence based on the likelihood ratio.
+All else equal, a lower Type 1 error rate means that observing a significant result is relatively more likely when H1 is true.
+Of course a higher likelihood ratio also means a higher Bayes factor, which means a higher probability that H1 is true in a Bayesian framework.
+
+> It is well known that the math for frequentists is similar or even identical to Bayesian under the right conditions.
+> Still, a frequentist would not conclude anything about the probability of H1.
+> However, our goal is to say something about how preregistration changes how believable researchers should find claims made in the literature.
+
+But the workhorse in the rationalization that the authors provide is the likelihood ratio.
+This does not change when we put a distribution on the Type 1 error rate (as the authors do).
+
+> Yes, the Bayes formula can be conceptualized as a likelihood ratio scaled by priors.
+> However, the mere use of the likelihood ratio in the Bayes theorem can not be grounds to reduce Bayesianism to likelihood ratios.
+
+The distribution (or the move to a Bayesian rationalization) does not add anything (although of course it can be added).
+
+> We find that it adds value by making an otherwise implicit process explicit to the point of calculatable.
+> By conceptualising uncertainty with a specific distribution, we rid ourselves of vague notions of transparency, control, or exploration.
+> This is of utmost importance because only this clarity enables us to effectively navigate the tradeoffs involved.
+
+Importantly, the likelihood function of course also underlies the uniformly most powerful test in a Neyman-Pearson hypothesis testing framework.
+So for a frequentist (and I mean a frequentist not in statistical terms, but also in Popperian anti-Bayesian view on confirmation) the same argument for preregistration plays a role.
+
+> We agree that this argument could perhaps be made for a frequentist as well.
+> However, it is not the math of the likelihood ratio that eludes us; it is how a significant result of a statistical test changes how a frequentist judges a hypothesis.
+> There is a great, yet unresolved, debate in frequentism about how to do just this.
+> In a Fisherian approach, no heat is paid to the power of a test; only the p-value is interpreted as the strength of evidence against the null.
```

Comment:
Would it make sense to put an overview over these different philosophical positions in the manuscript?